examples/docker/README.rst

```diff
@@ -50,6 +50,13 @@ matches the name of the image referenced in the ``MLproject`` file.
 
 Finally, run the example project using ``mlflow run examples/docker -P alpha=0.5``.
 
+.. note::
+    If running this example on M1 / M2 Mac, ensure that Docker Desktop is running and that you are
+    logged in to the Docker service. It is recommended to run this example with the optional
+    'Rosetta' compatibility mode (configured in settings) to ensure that if you make modifications
+    to scikit-learn versions that the appropriate cython compiler will be used to build
+    scikit-learn without errors.
```

Comment:
Does scikit-learn prvoide pre-compiled wheels for M1 / M2 Mac?

Reply:
The compatibility was introduced with commit #21827 for release version 1.0.2 all versions after that Dec 2021 release work just fine on M1/M2 OSX. Prior to that, there was no build for scikit-learn that would work properly (the only option was to install a large list of dependencies, ensure scipy versions were compatible, and configure gcc / blas packages to the correct build configuration for a given hardware version (year of release of M1 or M2 mac) and then compile it manually). This is a LOT of work for just an example to be functional for a 2.5 year old version of scikit-learn on our Docker examples.

Reply:
I'd remove this note if scikit-learn >= 1.0.2 works fine, but am ok with including this note.